cluster/addons/pod-resizer/Makefile

```diff
@@ -0,0 +1,42 @@
+# The build rule builds a Docker image that resizes its Kubernetes pod as
```

Comment:
Rather than describing the addon here, please add a README.md file in the `pod-resizer` directory and explain the purpose there. See https://github.com/kubernetes/kubernetes/blob/master/cluster/addons/dns/README.md for an example.